exchanges/btse/btse_websocket.go

```diff
@@ -94,12 +94,13 @@ func (b *BTSE) WsHandleData() {
 				}
 
 				b.Websocket.DataHandler <- wshandler.TickerData{
-					Timestamp:  time.Now(),
-					Pair:       currency.NewPairDelimiter(t.ProductID, "-"),
-					AssetType:  asset.Spot,
-					Exchange:   b.GetName(),
-					ClosePrice: price,
-					Quantity:   t.LastSize,
+					Exchange:  b.Name,
+					Close:     price,
+					Bid:       t.BestBids,
+					Ask:       t.BestAsk,
+					Last:      t.LastSize,
```

Comment:
Last size might be a volume variable.

Reply:
The old BTSE documentation is now gone, so I can't verify that. What I can verify is that it is always 0, so I'm removing it.

Reply:
:+1: